site/pages/pycon-thailand.en.rst

```diff
@@ -94,12 +94,25 @@ Sponsors
 
 .. container:: jumbotron clearfix
 
-   Our Sponsorship prospectus will be available soon.
+   Our Sponsorship prospectus is available for download now.
 
    .. raw:: html
 
+          <a class="btn btn-primary btn-lg active" href="/files/PyCon Thailand 2018 Sponsorship Prospectus.pdf">Download Prospectus</a>
           <a class="btn btn-primary btn-lg active" href="sponsorship">Register your interest</a>
 
+Gold Sponsors
```

Comment:
I think this can be combined with box above. so the box is for showing sponsors but while we need more it includes the button to encourage more sponsorship. later than turn into a smaller link

Reply:
btw, main sponsors also go in the footer (but smaller). platinum + gold